src/images.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+import { sendRequest } from './util';
+
+export default function images(): Object {
+  const baseUrl = 'images-api.nasa.gov';
+  const endpoints = {
+    search: '/search',
+    asset: '/asset',
+    metadata: '/metadata',
+    captions: '/captions',
+  };
+
+  function validateMediaType(mediaType: string): string {
```

Comment:
@AJFunk I'd suppose we could just add 'image,audio' and 'audio,image' to the array of valid media types?